### PR TITLE
AP-4403: Bump ZK to version 9.6

### PR DESCRIPTION
**Pull request by raboczi**
This PR bumps the ZK framework to version 9.6 and removes the custom Chiaroscuro Compact ZK theme in favor of the stock Iceblue Compact instead.  The Gradle configuration now sets the ZK version in one place.  This PR is a prerequisite for its ApromoreEE counterpart https://github.com/apromore/ApromoreEE/pull/617.

While the "Files changed" count looks alarming, it's mostly deleted files comprising the source code of Chiaroscuro Compact.

### CODE REVIEW

File: Apromore-Custom-Plugins/Process-Discoverer-Portal-Plugin/src/main/java/org/apromore/plugin/portal/processdiscoverer/components/CaseDetailsController.java [hunk 1]
```diff
@@ -180,10 +180,11 @@ public void onEvent(Event event) throws Exception {
         }
       });
       try {
-        org.zkoss.json.JSONObject param = (org.zkoss.json.JSONObject) event.getData();
+        // @todo Incorrect coordinate returned by ZK 9
```
Comment (Janite) [inline Apromore-Custom-Plugins/Process-Discoverer-Portal-Plugin/src/main/java/org/apromore/plugin/portal/processdiscoverer/components/CaseDetailsController.java:183]: Do we need to complete the todo task before merging this?

File: Apromore-Custom-Plugins/Process-Discoverer-Portal-Plugin/src/main/java/org/apromore/plugin/portal/processdiscoverer/components/CaseVariantDetailsController.java [hunk 2]
```diff
@@ -190,10 +190,13 @@ public void onEvent(Event event) throws Exception {
                 }
             });
             try {
-                org.zkoss.json.JSONObject param = (org.zkoss.json.JSONObject) event.getData();
+                // @todo Incorrect coordinate returned by ZK 9
```
Comment (Janite) [inline Apromore-Custom-Plugins/Process-Discoverer-Portal-Plugin/src/main/java/org/apromore/plugin/portal/processdiscoverer/components/CaseVariantDetailsController.java:193]: Same question as above (about todo)

File: Apromore-Custom-Plugins/Process-Discoverer-Portal-Plugin/src/main/java/org/apromore/plugin/portal/processdiscoverer/components/PerspectiveDetailsController.java [hunk 3]
```diff
@@ -102,10 +102,13 @@ public void onEvent(Event event) throws Exception {
             }); 
     
             try {
-                org.zkoss.json.JSONObject param = (org.zkoss.json.JSONObject) event.getData();
+                // @todo Incorrect coordinate returned by ZK 9
```
Comment (Janite) [inline Apromore-Custom-Plugins/Process-Discoverer-Portal-Plugin/src/main/java/org/apromore/plugin/portal/processdiscoverer/components/PerspectiveDetailsController.java:105]: Same question as above (about todo)

**Review by Janite — COMMENTED** [review]
There are a couple of todo comments about incorrect coordinates. Just wondering if that's something we're planning to fix in this PR or after ZK 9 is merged in?

**Review by dreamind — APPROVED** [review]

**Review by Janite — APPROVED** [review]
Looks good. There are a couple of minor UI issues/differences but nothing too big that we can't fix them after the merge. Listed them in here: https://apromore.atlassian.net/browse/AP-4671

**Pull request merged**